cmd/virtual-workspaces/main.go

```diff
@@ -33,17 +33,20 @@ import (
 )
 
 func main() {
-	stopCh := genericapiserver.SetupSignalHandler()
+	//signal handler
```

Comment:
nit: probably don't need the comment as `SetupSignalContext` is fairly self-explanatory.